.github/workflows/macos_debug_fetch_hwloc.yml

```diff
@@ -36,6 +36,7 @@ jobs:
               -DHPX_WITH_VERIFY_LOCKS=ON \
               -DHPX_WITH_VERIFY_LOCKS_BACKTRACE=ON \
               -DHPX_WITH_CHECK_MODULE_DEPENDENCIES=ON
+          ln -s "$(which aclocal)" /opt/homebrew/bin/aclocal-1.16
```

Comment:
Could you please extract this change into a separate PR?

Reply:
https://github.com/STEllAR-GROUP/hpx/pull/6628 here, thanks!